mysql/my.cnf

```diff
@@ -1,5 +1,8 @@
 [mysqld]
+user=mysql
+explicit_defaults_for_timestamp
```

Comment:
As far as I can see in the [phabricator repository](https://github.com/phacility/phabricator/search?l=sql&q=TIMESTAMP&type=Code&utf8=%E2%9C%93) they don't use TIMESTAMP columns.